src/utils/matmul_helper.h

```diff
@@ -480,6 +480,7 @@ class MMHelper {
     template <typename InT, typename WeiT, typename OutT>
     static void compute(bool transA, int M, int N, int K, float alpha, const InT *A, int lda, const WeiT *packedB,
             const float *scaleB, const float *zeroB, float beta, OutT *C, int ldc) {
+        Env::xft_set_verbose();
```

Comment:
Could call it once at the beginning of the application.

Reply:
https://github.com/intel/xFasterTransformer/blob/main/src/models/models.cpp#L102

Reply:
@changqi1 I revised it in models.h and models.cpp and kept Model::config as its original implement which was @Duyi-Wang 's strong suggestion.

Reply:
It should be called in constructor instead of model.config(), which will be called at each generation in python.

Reply:
Thanks Duyi, you also recommend keep two Model::config functions as its original implement.